doc/classes/Texture2DArray.xml

```diff
@@ -1,8 +1,20 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Texture2DArray" inherits="TextureLayered" version="4.0">
 	<brief_description>
+		A [TextureLayered] of type [code]enum TextureLayeredType.TEXTURE_LAYERED_2D_ARRAY[/code].
```

Comment:
This is tautological. It doesn't instruct a user who doesnt know what a 2D array texture is. 

Try something like "Composite texture type made of multiple images with same dimensions"

Reply:
should be better now:  
`A single texture resource which consists of multiple, separate images. Each image has the same dimensions and number of mipmap levels.`